requirements-dev.in

```diff
@@ -5,3 +5,4 @@ isort
 pytest
 pytest-cov
 pytest-django
+factory_boy
```

Comment:
didn't we agree to use hyphens instead of underscores? 😛

Reply:
:D crap